### PR TITLE
eris: add migaloo tvl

### PR DESCRIPTION
**Pull request by 0xphilipp**
This PR adds support for ERIS TVL on Migaloo chain (Cosmos).

Do you need more infos on the newly added chain Migaloo chain?

Main Token: https://www.coingecko.com/en/coins/white-whale

migaloo

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/eris-protocol exports TVL: 
        
 
 
    terra2                    1.46 M
    kujira                    875.98 k
    juno                      1.63 k
    terra                     46.00 k
    migaloo                   1.31 M
    
    total                    3.69 M

File: projects/helper/chain/cosmos.js [hunk 1]
```diff
@@ -205,6 +209,7 @@ module.exports = {
   endPoints,
   totalSupply,
   getBalance,
+  getBalance2,
```
Comment (0xphilipp) [inline projects/helper/chain/cosmos.js:211]: this is the only change, the rest is formatting

**Comment by llamatester** [comment]
The adapter at projects/eris-protocol exports TVL: 
        
 
 
    kujira                    875.98 k
    terra2                    1.46 M
    juno                      1.63 k
    terra                     46.00 k
    migaloo                   1.31 M
    
    total                    3.69 M

**Comment by g1nt0ki** [comment]
@0xphilipp looks good, thanks for the PR

**Pull request merged**